.circleci/config.yml

```diff
@@ -396,7 +436,7 @@ orbs:
         steps:
           - run:
               name: Wait for container on port <<parameters.port>>
```

Comment:
We could `<<parameters.host>>` in the name as well

Reply:
Yup, added this